omod/src/main/webapp/resources/htmlforms/Covid19-ReferralForm.xml

```diff
@@ -0,0 +1,376 @@
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div class="tab-pane active" id="patient-history">
+                                <div class="card">
+                                    <div class="card-header">
+                                            Patient History
+                                        </div>
+                                    <div class="card-body">
+                                        <div class="row">
+                                            <div class="col-md-4">
+                                                <label>Chronic Condition/PMH</label>
+                                                <obs conceptId="159395" />
+                                            </div>
+                                            <div class="col-md-4">
+                                                <label>Presenting Complaint and History</label>
+                                                <obs conceptId="159395" style="textarea" />
+                                            </div>
+                                            <div class="col-md-4">
+                                                <label>Chronic Medication</label>
+                                                <obs conceptId="159395" style="textarea" />
+                                            </div>
+                                        </div>
+                                        <div class="row">
+                                            <div class="col-md-4">
+                                                <label>History of Current Illness</label>
+                                                <obs conceptId="159395" style="textarea" />
+                                            </div>
+                                            <div class="col-md-8">
+                                                <label>Social History</label>
+                                                <div class="row">
+                                                    <obs id="smoking" conceptId="166351" answerConceptId="166350" answerLabel="Smoking" style="checkbox" class="col-md-3" />
+                                                    <obs id="alcohol" conceptId="166351" answerConceptId="794" answerLabel="Alcohol" style="checkbox" class="col-md-3" />
+                                                    <obs id="illicit-drug-use" conceptId="166351" answerConceptId="166349" answerLabel="Illicit Drug Use" style="checkbox" class="col-md-3" />
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div class="tab-pane active" id="examinations">
```

Comment:
Remove "active" class among classes. this is causing the tab content  on load to be in the first tab